pennylane/tape/tape.py

```diff
@@ -462,7 +462,7 @@ def _process_queue(self):
             if isinstance(obj, QuantumTape):
                 self._ops.append(obj)
 
-            elif isinstance(obj, qml.operation.Operation) and not info.get("owner", False):
+            elif isinstance(obj, qml.operation.Operator) and not info.get("owner", False):
```

Comment:
Nice, so that is the line that is now allowed with your changes, right?

Reply:
I thought that was going to be the extent of this PR; obviously it was not.  Changing this requires all of those other changes.

We can either make the wrappers inherit from `Operation` (keeps inheritance structure specific logic and will probably cause problems later down the road) or use the dispatch on instance property as proposed in #2408 .

Seems like there's complaints about the solution in #2408.  But that would be cleaner than this PR.